apps/librepcb-cli/commandlineinterface.cpp

```diff
@@ -90,8 +90,9 @@ int CommandLineInterface::execute() noexcept {
   // option which we don't need, and the OS-dependent option "-?".
   const QCommandLineOption helpOption({"h", "help"}, tr("Print this message."));
   parser.addOption(helpOption);
-  const QCommandLineOption versionOption = parser.addVersionOption();
-  QCommandLineOption verboseOption("verbose", tr("Verbose output."));
+  const QCommandLineOption versionOption({"V","version"},tr("Show Version.")); // = parser.addVersionOption();
```

Comment:
I'd make the "Version" lowercase for consistency with the rest of the help text. And let's remove the comment to keep it clean :see_no_evil: 

```suggestion
  const QCommandLineOption versionOption({"V","version"},tr("Show version."));
```